src/app/shared/components/card/icon/icon.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-icon',
+  templateUrl: './icon.component.html',
+  styleUrls: ['./icon.component.scss'],
+})
+export class IconComponent {
+  @Input() icon: string = '';
```

Comment:
Não precisa tipar como string se icon sempre for uma string, já fica inferido devido a igualdade.